crates/ruff_python_ast/ast.toml

```diff
@@ -77,8 +86,7 @@ fields = [
   { name = "name", type = "Identifier" },
   { name = "type_params", type = "Box<crate::TypeParams>?" },
   { name = "parameters", type = "Box<crate::Parameters>" },
-
-  { name = "returns", type = "Expr?" },
+  { name = "returns", type = "Expr?", is_annotation = true },
```

Comment:
What's the meaning of `in_annotation`. Could we add some documentation for it.

Reply:
Right, I forgot about this one. This is only needed to determine the visitor function. For other expressions we use `visit_expr` but for annotations we have `visit_annotation`.

Reply:
My preference is to preserve the behavior from the hand written visitor and to change the visiting (if that's indeed the case) in a separate PR